server/src/main/java/io/crate/auth/JWTAuthenticationMethod.java

```diff
@@ -130,27 +129,8 @@ public JWTAuthenticationMethod(Roles roles,
         return user;
     }
 
-
-    public static JwkProvider jwkProvider(@NotNull String issuer) {
```

Comment:
The custom JwkProvider does not have the logic of `WELL_KNOWN_JWKS_PATH` therefore we can remove this.